auth/src/main/java/com/firebase/ui/auth/ui/FlowParameters.java

```diff
@@ -51,21 +51,36 @@
 
     public final boolean allowNewEmailAccounts;
 
+    public final boolean isReauth;
```

Comment:
@amandle Couldn't this be a method instead? It seems weird to have a parameter just for something that can be inferred programmatically.

Something like:
```java
public boolean isReauth() {
    return !TextUtils.isEmpty(reauthEmail);
}
```